tutorials/applications/Spin chain of 3 atoms in XY mode.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<p>Why is this angle supressing the interaction? Explain that it depends on the square of the cosine and that this is the orientation that cancels the term.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I would like to add a figure. May I create a folder <code>figures</code> in  the folder <code>tutorials</code>?</p><p> <span class="ql-cursor">﻿</span></p>

Reply:
We are including figures in the `docs/source/files` folder, so that the online documentation is also rendered. Include it in the notebooks like this example:

```html
<img src="../../docs/source/files/charles_bronson.png" alt="The Stone Killer" style="width: 500px"/>
```

Reply:
<p>Thanks</p>